routes/volunteering_events.js

```diff
@@ -1,7 +1,28 @@
 const Router = require('koa-router');
+const VolunteeringEvent = require('../models/VolunteeringEvent');
+const DBMethods = require('./baseDBMethods');
+const VolunteeringOfferRequestBaseModel = require('../models/VolunteeringOfferRequestBase');
 
 const router = new Router();
 
-// Add sub-routes here using router.get(...), router.post(...), etc.
+function getOfferRequestOfEvent(fieldName) {
```

Comment:
`getOfferRequestOfEvent` -> `getRequestOrOffer`
Because "of event" is redundant here, since everything here is "of event".
(Also, request-offer and offer-request doesn't really matter, but request-offer is the order we used so far)

Reply:
?